server/.env.example

```
@@ -12,38 +12,106 @@ DB_NAME=akash_vpn
+BELGIUM_L2TP_PHASE1_ALGORITHMS=aes128-sha1-modp2048
+BELGIUM_L2TP_PHASE2_ALGORITHMS=aes128-sha1
+BELGIUM_CA_CERT=...
+BELGIUM_V2RAY_UUID=...
+BELGIUM_V2RAY_PORT=30419
+
+
+# Australia Server Configuration
+AUSTRALIA_HOSTNAME=https://host
+AUSTRALIA_PORT=31975
+AUSTRALIA_USERNAME=username
+AUSTRALIA_PASSWORD=password
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AUSTRALIA_PASSWORD key should go before the AUSTRALIA_PORT key

(UnorderedKey)